### PR TITLE
add stories for input & textarea

### PR DESCRIPTION
**Pull request by rahkarino**

### CODE REVIEW

File: src/components/input/Input.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { Input, TInput } from "./Input";
+
+export default {
+  title: "Components/Input",
+  component: Input,
+} as ComponentMeta<typeof Input>;
+
+const Template: ComponentStory<typeof Input> = (args) => <Input {...args} />;
```
Comment (amirHosseinKhademii) [inline src/components/input/Input.stories.tsx:10]: We need to create storybook from dummy component

File: src/components/input/Input.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Input as AntInput, Form } from "antd";
 import { ComponentProps } from "react";
 import { useController } from "react-hook-form";
 
-type TInput<G> = Omit<ComponentProps<"input">, "size" | "ref" | "name"> & {
+export type TInput<G> = Omit<ComponentProps<"input">, "size" | "ref" | "name"> & {
```
Comment (amirHosseinKhademii) [inline src/components/input/Input.tsx]: We need to rename this to form input and create a dummy input in seperate file and create storybook base on that

File: src/components/text-area/TextArea.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { TextArea, TTextarea } from "./TextArea";
+
+export default {
+  title: "Components/TextArea",
+  component: TextArea,
+} as ComponentMeta<typeof TextArea>;
+
+const Template: ComponentStory<typeof TextArea> = (args) => (
```
Comment (amirHosseinKhademii) [inline src/components/text-area/TextArea.stories.tsx]: Same as input

File: src/components/text-area/TextArea.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { Form, Input } from "antd";
 import { ComponentProps } from "react";
 import { useController } from "react-hook-form";
 
-type TTextarea<G> = Omit<
+export type TTextarea<G> = Omit<
```
Comment (amirHosseinKhademii) [inline src/components/text-area/TextArea.tsx]: Same as form input

File: src/components/input/Input.tsx (hunk 2 above)
Comment (rahkarino) [reply src/components/input/Input.tsx]: done!

File: src/components/text-area/TextArea.stories.tsx (hunk 3 above)
Comment (rahkarino) [reply src/components/text-area/TextArea.stories.tsx]: done!

File: src/components/text-area/TextArea.tsx (hunk 4 above)
Comment (rahkarino) [reply src/components/text-area/TextArea.tsx]: done!

**Review by amirHosseinKhademii — APPROVED** [review]

**Pull request merged**